cookiecutter.json

```diff
@@ -1,5 +1,9 @@
 {
     "project_name": "sugardough",
+    "name": "Sugardough",
```

Comment:
You end up using `project_name` in `contribute.json` which is good. This is no longer needed.